src/core/toga/app.py

```diff
@@ -226,6 +273,11 @@ def __init__(
         self._startup_method = startup
 
         self._main_window = None
+        # In this world, TogaApp.windows would be a set-like object
+        # that has add/remove methods (including support for
+        # the + and += operators); adding a window to TogaApp.windows
+        # would assign the window to the app.
+        self.windows = Windows(self, windows)
```

Comment:
To be clear - `self.windows` makes sense as a variable name; it's the data type (`Windows`) we should change.